lib/Doctrine/Annotations/Annotation.php

```diff
@@ -14,8 +14,6 @@ class Annotation
 {
     /**
      * Value property. Common among all derived classes.
-     *
-     * @var string
```

Comment:
Should be annotated as `mixed` instead of being skipped

Reply:
Isn't `mixed` implied when it's not further specified?

Reply:
This was also my assumption. Unless we're explicitly saying `mixed` just to show we care enough :D

Reply:
> Isn't `mixed` implied when it's not further specified?

"implied" is something we no longer really allow. Even our coding standards prevent us from skipping parameters/return types. Possibly a missing CS rule, @carusogabriel?

Reply:
@Ocramius I’ll add some tests in our CS repo and discovered what should be fixed to avoid it :)